### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by PaulKovalov**
Here I add a few features like dark mode and scroll to top as well as improve styling of the page

### CODE REVIEW

**Comment by NeilLi1992** [comment]
Hmm it's a bit confusing as the content was reviewed in PR #3 yesterday, but they appear in the diff again. Then I realised #3 was merged into branch `page-skeleton`. Was this intentional? Can we merge all PRs into master?

**Comment by NeilLi1992** [comment]
I've changed the base to 'page-skeleton' so it's more clear to see the diff since last review. I suggest create an extra merge from 'page-skeleton' to 'master', and for future PRs always merge into master.

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -54,3 +54,17 @@ function updateButtonsState(visibleImgId) {
     nextButton.disabled = false;
   }
 }
+
+function toggleTheme() {
+  var toggler = document.getElementById('theme-toggler');
+  var body = document.getElementsByTagName('body')[0];
```
Comment (NeilLi1992) [inline portfolio/src/main/webapp/script.js:60]: Same as in other functions, it's a good practice to use `const` by default, unless when you actually when to mutate the variable, then change to `var`.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -54,3 +54,17 @@ function updateButtonsState(visibleImgId) {
...
+  if (toggler.checked) {
+    body.classList.remove('light-theme');
+    body.classList.add('dark-theme');
+  } else {
+    body.classList.remove('dark-theme');
+    body.classList.add('light-theme');
+  }
+}
+
+function scrollUp() { window.scrollTo({top : 0, behavior : "smooth"}); }
```
Comment (NeilLi1992) [inline portfolio/src/main/webapp/script.js:70]: Usually this should have been handled by linter I guess, but it's visually more clear to separate function body to new lines, even it's a one-line function.

**Comment by PaulKovalov** [comment]
> Hmm it's a bit confusing as the content was reviewed in PR #3 yesterday, but they appear in the diff again. Then I realised #3 was merged into branch `page-skeleton`. Was this intentional? Can we merge all PRs into master?

Sorry, it is my mistake, I had to change the base branch back to master when merging `page-skeleton`

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (PaulKovalov) [reply portfolio/src/main/webapp/script.js:70]: It was actually edited by clang linter, which is recommended by Google stylesheet as the best for JavaScript. Probably I need to set it up a bit

**Comment by PaulKovalov** [comment]
> I've changed the base to 'page-skeleton' so it's more clear to see the diff since last review. I suggest creating an extra merge from 'page-skeleton' to 'master', and for future PRs always merge into master.

Fixed it, changing base back to master

**Pull request merged**